src/ngraph/runtime/dynamic/dynamic_backend.cpp

```diff
@@ -174,6 +175,7 @@ bool runtime::dynamic::DynamicExecutable::call(
     }
 
     pass::Manager passes;
+    passes.register_pass<pass::Opset0Downgrade>();
```

Comment:
In current version for dynamic ops `Opset0Downgrade` is performed twice (firstly for `dynamic shape` backend and secondly for inner `static`). Any idea how to deal with it?

Reply:
What if some other backend would like to get and opset1 instead of opset0?
example:
1. Produce opset1 operators.
2. Downgrade operators to opset0
3. Pass these operators to backend which require opset1.
4. Upgrade operators to opset1
5. Run computation

This flow looks invalid IMHO.

Reply:
@jbobba @diyessi Have you idea how to handle such scenario?

Reply:
In current state passes flow in `dynamic_backend`:
- `ConstantFolding`
- `Opset0Downgrade`
- `DynElimination`
works both for `Broadcast:v1`, `Reshape:v1` and  `StridedSlice:v1`
@jbobba Is it good solution?

Reply:
@mbencer We should make these upgrade/downgrade passes more configurable so backends can decide which ops to downgrade/upgrade. Something similar to FusedOpDecomposition.